pkgs/top-level/all-packages.nix

```diff
@@ -6178,6 +6178,7 @@ with pkgs;
 
   onnxruntime = callPackage ../development/libraries/onnxruntime {
     inherit (darwin.apple_sdk.frameworks) Foundation;
+    inherit (pkgs.config) cudaSupport;
```

Comment:
Redundant if there's `config, cudaSupport ? config.cudaSupport` in the arguments

Reply:
Thank you, fixed